documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
 
 This is useful if you are really concerned about limiting the data you expose, but provides the least utility for the consumer depending on the activity they are trying to test.
 
 ### Profile: moderate
 
-The `moderate` profile attempts to be a bit smarter about what it anonymizes and is the recommended one for most cases. It operates different depending on the data type:
+The `moderate` profile attempts to be a bit smarter about what it anonymizes and is **the recommended profile for most cases**. It sorts and anonymizes data by data type:
 
-* node name: is anonymized by default always
-* resource type name: the core types that are built-in to Puppet are not anonymized, including some common types from the modules: stdlib, postgresql, rabbitmq, puppetdb, apache, mrep, f5, apt, registry, concat and mysql. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type names as they may contain confidential information.
-* resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service and Stage.
-* parameter names: are never anonymized
-* parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit` and `schedule`.
-* report log messages: are always anonymized
-* file names: are always anonymized
-* file numbers: are left as they are
-* log messages: are always anonymized
-* metrics: metric names in the `time` category are anonymized as resource
-  types.
+* Node name: is anonymized by default.
+* Resource type name: the core types that are built into Puppet are not anonymized, including some common types from the modules: `stdlib`, `postgresql`, `rabbitmq`, `puppetdb`, `apache`, `mrep`, `f5`, `apt`, `registry`, `concat`, and `mysql`. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type name, as these may contain confidential information.
+* Resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service, and Stage.
+* Parameter names: are never anonymized.
+* Parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit`, and `schedule`.
+* Report log messages: are always anonymized.
+* File names: are always anonymized.
+* File numbers: are left as they are.
+* Log messages: are always anonymized.
+* Metrics: metric names in the `time` category are anonymized as resource types.
 
 ### Profile: low
 
-This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. By default most things are not anonymized except for:
+This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. However, by default, the following categories are annonymized:
```

Comment:
anonymized, and I'd also just start the last sentence at the "the". Once the profile is chosen there's not really a non-default case.